includes/class-ssp-settings.php

```diff
@@ -1179,6 +1261,9 @@ public function display_field( $args ) {
 			case 'number':
 				$html .= '<input id="' . esc_attr( $field['id'] ) . '" type="' . $field['type'] . '" name="' . esc_attr( $option_name ) . '" placeholder="' . esc_attr( $field['placeholder'] ) . '" value="' . esc_attr( $data ) . '" class="' . $class . '"/>' . "\n";
 				break;
+            case 'colour-picker':
+                $html .= '<input id="' . esc_attr( $field['id'] ) . '" type="' . $field['type'] . '" name="' . esc_attr( $option_name ) . '" placeholder="' . esc_attr( $field['placeholder'] ) . '" value="' . esc_attr( $data ) . '" class="' . $class . '"/>' . "\n";
```

Comment:
Following error on Podcast settings page 
Notice: Undefined index: placeholder in /var/www/public/wp-content/plugins/seriously-simple-podcasting/includes/class-ssp-settings.php on line 1265

Reply:
@TheCraigHewitt does it make sense to hide the html5 player settings and only show them when that player style is selected?